index.html

```diff
@@ -21,14 +16,27 @@
     </style>
     <script
       type="text/javascript"
-      src="https://timbl.com/timbl/Automation/Library/Mashup/mashlib-prealpha.js"
+      src="http://solid.github.io/mashlib/dist/mashlib.js"
```

Comment:
Would it make sense to set up bundle for this version instead of relying on a published version of mashlib?

Reply:
Yes! So then this repo needs a build script.
And then it should be requiring solid-ui, not mashlib, see https://github.com/inrupt/data-browser-docs/blob/master/docs/current.md#technical-model

Reply:
Hmmm   ... one advantage of having a common URI for hosted mashlib.js is the caching of the library by browsers when users use more than one app.

Reply:
Right, we should tell our various webpack configs to fetch rdflib.js and solid-ui from cdn, see https://webpack.js.org/configuration/externals/. I created #10 about that.